packages/nestjs-common/src/domain/role/interfaces/role-ownable.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { ReferenceId } from '../../../reference/interfaces/reference.types';
+import { RoleInterface } from './role.interface';
+
+export interface RoleOwnableInterface {
+  roleId?: ReferenceId;
```

Comment:
roleId property should be required
```suggestion
  roleId: ReferenceId;
```